package proto_ff;

import "yd_fieldoptions.proto";


message conditionconditionconditionDesc
{
	optional int32 Type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 Param = 2[(yd_fieldoptions.field_cname) = "参数"];
}

message conditioncondition
{
	optional int32 id = 1[(yd_fieldoptions.field_cname) = "id"];
	repeated conditionconditionconditionDesc condition = 2[(yd_fieldoptions.field_cname) = "条件", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_conditioncondition
{
	repeated conditioncondition conditioncondition_List = 1[(yd_fieldoptions.field_arysize)=125];
}
